proto/file.proto

```diff
@@ -82,6 +83,13 @@ message RemoveReply {
     optional Error error = 1;
 }
 
+message SyncRequest {
+    repeated string names = 1;
```

Comment:
why do we use `repeated` instead of `optional` here? Do we want to support sync multiple files at the beginning?